docs/storage-abstraction.md

```diff
@@ -584,21 +523,26 @@ Scan scan =
         .limit(10)
         .build();
 
-// Execute the Scan operation
+// Execute the `Scan` operation.
 Scanner scanner = storage.scan(scan);
 ```
 
-Note that you can't specify clustering key boundaries and orderings in Scan without a partition key.
+{% capture notice--info %}
+**Note**
 
-#### Put operation
+You can't specify clustering-key boundaries and orderings in `Scan` without specifying a partition key.
+{% endcapture %}
 
-`Put` is an operation to put a record specified by a primary key.
-It behaves as an upsert operation: If it exists, it updates it; otherwise, it inserts it.
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-You need to create a Put object first, and then you can execute it with the `storage.put()` method as follows:
+#### `Put` operation
+
+`Put` is an operation to put a record specified by a primary key. The operation behaves as an upsert operation for a record, in which the operation updates the record if the record exists or inserts the record if the record does not exist.
```

Comment:
In PR #1084 that contains revisions for the [`api-guide.md`](https://github.com/scalar-labs/scalardb/blob/6f221b039243262a061aabd36f7652a17fcb5f75/docs/api-guide.md?plain=1#L657), we have a note after this paragraph. Do we need that same note here?: 

```suggestion
`Put` is an operation to put a record specified by a primary key. The operation behaves as an upsert operation for a record, in which the operation updates the record if the record exists or inserts the record if the record does not exist.

{% capture notice--info %}
**Note**

When you update an existing record, you need to read the record by using `Get` or `Scan` before using a `Put` operation.
{% endcapture %}

<div class="notice--info">{{ notice--info | markdownify }}</div>
```

Reply:
No, we don't need the note for the storage API. It's only for the transaction API.